### PR TITLE
docs: update main example

### PR DESCRIPTION
**Pull request by OrKoN**
And sync README description

### CODE REVIEW

**Review by Lightning00Blade — COMMENTED** [review]
I am not sure about changing the example.

For me it's a little bit confusing.

**Comment by OrKoN** [comment]
> I am not sure about changing the example.
> 
> For me it's a little bit confusing.

could you be more specific?

**Comment by OrKoN** [comment]
Extracted description update to https://github.com/puppeteer/puppeteer/pull/12899

File: README.md [hunk 1]
```diff
@@ -16,39 +16,58 @@
...
+const chrome = await puppeteer.launch({
+  browser: 'chrome',
+});
 
-// Navigate the page to a URL.
-await page.goto('https://developer.chrome.com/');
+// Launch Firefox.
+const firefox = await puppeteer.launch({
+  browser: 'firefox',
+});
```
Comment (matthiasrohmer) [inline README.md:35]: I wonder if we should make both use BiDi...? Can't decide what's the best message to send: you can do all the things equally in Firefox and Chrome using CDP and BiDi *or* look you can do the same things with the same protocol in both browsers.

I am leaning towards the later.

**Review by matthiasrohmer — APPROVED** [review]
Thanks for updating Alex! I think this is a good enough start for now - I'll start working on the homepage update next week and then think of probably 3 specific examples that highlight certain automations.

One thought, that's going nowhere: automating the visit to a site that's different in Firefox and Chrome could be fun, like whatismybrowser.com - but using a 3P site as an example is bad. And I am saying this as the person formerly being responsible for d.c.c's search and constantly wondering about the amount of queries for `automate beyond recorder` :)

**Pull request closed without merging**